scripts/test-cov.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 set -e
 
+node="yarn node"
 jestArgs="--coverage"
 
 if [ -n "$CI" ]; then
   jestArgs="${jestArgs} --maxWorkers=4 --ci"
 fi
 
-node_modules/.bin/jest $jestArgs
+$node node_modules/.bin/jest $jestArgs
```

Comment:
We leave `jest` to be relative path as we plan to unplug jest on yarn 2 so that it can work with `--inspect-brk`.